test/MyFirstApiConsume.test.js

```diff
@@ -0,0 +1,91 @@
+const axios = require('axios');
+const { expect } = require('chai');
+const { StatusCodes } = require('http-status-codes');
+
+describe('First Api Tests', () => {
+
+    it('Consume GET', async () => {
+        const response = await axios.get('https://httpbin.org/ip');
+
+        expect(response.status).to.equal(StatusCodes.OK);
+        expect(response.data).to.have.property('origin');
+    });
+
+    it('Consume GET Service with query parameters', async () => {
+        const query = {
+            name: 'John',
+            age: '31',
+            city: 'New York'
+        };
+
+        const response = await axios.get('https://httpbin.org/get', { query });
+
+        expect(response.status).to.equal(StatusCodes.OK);
+        expect(response.config.query).to.eql(query);
+    });
+
+    //Get only headers
+    it('consume HEAD', async () => {
+        const response = await axios.head('https://httpbin.org/headers');
+
+        expect(response.status).to.equal(StatusCodes.OK);
+        expect(response.headers).to.have.property('content-type', 'application/json');
+    });
+
+    //Partial updates
+    it('Consume PATCH', async () => {
+        const body = {
+            name: 'Maria',
+            lastname: 'Sierra',
+            age: '30',
+            city: 'Cartagena'
+        };
+
+        const response = await axios.patch('https://httpbin.org/patch', body);
+
+        expect(response.status).to.equal(StatusCodes.OK);
+        expect(response.data.json).to.eql(body);
+    });
+
+    //Replace the resource in its entirety
+    it('Consume PUT', async () => {
+        const body = {
+            name: 'Maria',
+            lastname: 'Sierra',
+            age: '30',
+            city: 'Cartagena'
+        };
+
+        const response = await axios.put('https://httpbin.org/put', body);
+
+        expect(response.status).to.equal(StatusCodes.OK);
+        expect(response.data.json).to.eql(body);
+    });
+
+    it('Consume DELETE', async () => {
+        const body = {
+            name: 'Maria',
+            lastname: 'Sierra',
+            age: '30',
+            city: 'Cartagena'
+        };
+    
+        const response = await axios.delete('https://httpbin.org/delete', body);
+        expect(response.status).to.equal(StatusCodes.OK);
+    });
+
+    it('Consume POST', async () => {
+        const body = {
+            name: 'Maria',
+            lastname: 'Sierra',
+            age: '30',
+            city: 'Cartagena'
+        };
+
+        const response = await axios.post('https://httpbin.org/post', body);
+
+        expect(response.status).to.equal(StatusCodes.OK);
+        expect(response.data.json).to.eql(body);
+    });
+
+});
```

Comment:
EOF

Reply:
![Captura de pantalla 2022-06-23 210808](https://user-images.githubusercontent.com/90917883/175448081-cac304ce-e49b-4a6b-9557-6cb41d2b7af1.png)

Reply:
Remember adding a break line at the end of the file to avoid the EOF warning 
![image](https://user-images.githubusercontent.com/34688953/175448474-ce0fc374-50d5-4f35-bf6f-aa9b20565057.png)

You can configure Visual Studio Code to do it automatically https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files

Reply:
Thanks!!